LocalLang/Typing.lean

```diff
@@ -5,13 +5,21 @@ import LocalLang.Types
 
 mutual
   inductive TypeJdg : TypeContext → Expr → LLType → Prop where
-    | jdgVar {Γ : TypeContext}  {name : String} (H : (name, LLType.nat) ∈ Γ)
-                      : TypeJdg Γ (.var name) LLType.nat
+    | jdgVar {Γ : TypeContext}  {name : String} {ty : LLType} (H : Γ[name]? = some ty)
```

Comment:
Could you change Prop-valued names to be snake_case please? For constructors and assumptions both.